Source/Redemption/Public/StaminaBarWidget.h

```diff
@@ -0,0 +1,28 @@
+// Fill out your copyright notice in the Description page of Project Settings.
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "ParryBarWidget.h"
+#include "StaminaBarWidget.generated.h"
+
+/**
+ * 
+ */
+UCLASS()
+class REDEMPTION_API UStaminaBarWidget : public UParryBarWidget
+{
+	GENERATED_BODY()
+	
+public:
+	UPROPERTY(EditAnywhere)
+	FColor FullStamina;
+
+	UPROPERTY(EditAnywhere)
+	FColor LowStamina;
+
```

Comment:
since it is a child of parrybar, we could just have a variable called FullBar and LowBar, instead of FullParry, FullStamina, LowParry, LowStamina

Reply:
Okay, thank you for advice :)